.github/workflows/publish_linux_release_candidate.yml

```diff
@@ -0,0 +1,70 @@
+name: Publish Linux Release Candidate
+
+on: 
+  workflow_dispatch:
+    inputs:
+      commit_sha:
+        description: "SHA of commit to publish (example: 2ac8bd4a915a7c8a3e3a63081d72da33baa9ea7e)"
+        required: true
+
+jobs:
+  release: 
+    env:
+      RELEASES_REPOSITORY: ${{ secrets.RELEASES_REPOSITORY }}
+      GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+      # STORE_LOGIN: ${{ secrets.SNAP_STORE_LOGIN }} 
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with: 
+        commit: ${{ github.event.inputs.commit_sha }}
+
+    - name: Install Node
+      uses: actions/setup-node@v2.2.0
+      with:
+        node-version: 16
+
+    - name: Install NPM Dependencies
+      run: npm ci
+
+    - name: Tag Release Commit
+      run: |
+        git config --global user.name "OutlineBot"
+        git config --global user.email "outlinebot@users.noreply.github.com"
+
+        git tag "linux-$(node scripts/get_version.mjs linux)"
+
+    - name: Download Release Candidate
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: build_release_candidates.yml
+        commit: ${{ github.event.inputs.commit_sha }}
+        name: release-linux-${{ github.event.inputs.commit_sha }}
+
+    - name: Checkout Releases Repository
+      uses: actions/checkout@v2.3.4
+      with:
+        repository: ${{ env.RELEASES_REPOSITORY }}
+        token: ${{ env.GITHUB_TOKEN }}
+        path: outline-releases
+    
+    - name: Push to Releases Repository
+      run: |
+        cp build/dist/Outline-Client.AppImage outline-releases/client/stable/Outline-Client.AppImage
+        cp build/dist/linux-latest.yml outline-releases/client/stable/linux-latest.yml
+        RELEASE_VERSION="$(node scripts/get_version.mjs linux)"
+        cd outline-releases
+        git add client/stable/Outline-Client.AppImage client/stable/linux-latest.yml
+        git commit -m "Release linux client ${RELEASE_VERSION}"
+        git tag "linux-${RELEASE_VERSION}"
+        git push origin master
+
+    # TODO(daniellacosse): S3 bucket?
```

Comment:
just want to make sure this S3 bucket is still a deploy target - https://github.com/Jigsaw-Code/outline-releases/blob/master/.github/workflows/upload_linux_client_to_s3.yml